### DISCUSSION TITLE
GitHub wiki document up-to-date?

### DISCUSSION THREAD

**Discussion by abhipsaMisra** [dotnet/MQTTnet#1744] [Q&A] [answered]
Question regarding the [wiki](https://github.com/dotnet/MQTTnet/wiki/MQTT-Version-5-Features) doc on this repo - is it up to date? I see a couple of features marked as "not supported" in the document wrt MQTT 5 support, however, I can see code referencing said feature.

Is the wiki something that can be used as a reference while investigating the MQTT 5 feature matrix support, or do I need to look at the implementation directly?

Eg: shared subscriptions are listed as "not supported" on the wiki, however, the connect result packet does inform the client if the broker supports shared subscription.
Does this mean that shared subscriptions are supported directly by the client using MqttNet, or is this only an "fyi" regarding the broker's behavior?

**Comment by SeppPenner** [comment, marked as answer]
The wiki is not up-to-date at all and will not be updated anymore. If you check the Examples page under https://github.com/dotnet/MQTTnet/wiki/Examples --> There it's stated that only v3.x is covered in the wiki.
